dashboard/db/schema.rb

```diff
@@ -742,6 +742,8 @@
     t.integer  "script_id",       limit: 4
     t.integer  "level_source_id", limit: 4
     t.boolean  "submitted"
+    t.boolean  "view_answers"
```

Comment:
You could use [ActiveRecord::Enum](http://api.rubyonrails.org/v4.1/classes/ActiveRecord/Enum.html) and add a `submitted` method that wraps `status.submitted?` or whatever the enum will be called.